infrastructure/Taskfile.yml

```
@@ -223,6 +223,36 @@ tasks:
       cmds:
         - task/scripts/promote-workloads-to-prod-nodes.sh
 
+    cluster:mode:release:
+      deps: [cluster:auth , lagoon:cli:config]
+      desc: "Set the cluster in release mode by adding additional nodes to release critical node pools"
+      cmds:
+        - az aks nodepool scale
+          --cluster-name aks-dplplat01-01
+          --nodepool-name app6
+          --resource-group rg-env-dplplat01
+          --node-count 4
+        - az aks nodepool scale
+          --cluster-name aks-dplplat01-01
+          --nodepool-name system2
+          --resource-group rg-env-dplplat01
+          --node-count 5
+
+    cluster:mode:business:
+      deps: [cluster:auth , lagoon:cli:config]
+      desc: "Set the cluster in release mode by adding additional nodes to release critical node pools"
```

```suggestion
    cluster:mode:reset:
      deps: [cluster:auth , lagoon:cli:config]
      desc: "Reset changes to the cluster mode returning it to standard operations."
```

The description looks like copy/paste and should be changed :smile:.

Changing the name is just a suggestion.

---

Thats a good suggestion - I was hoping someone would come up with a suggestion :)